proto/grid/service/signaling_service.proto

```diff
@@ -8,11 +8,38 @@ import "proto/core/io/address.proto";
 message SignalingAnswerMessage {
   syft.core.common.UID msg_id = 1;
```

Comment:
Q: Is the ```msg_id``` different than the ```Id``` we use for objects?

Reply:
No it shouldn't be - Message subclasses from ObjectWithId and they should be the same.

Reply:
(although in places we can't just call it "id" because there are other ids around)

Reply:
I'm not sure how it works internally.
I just followed the pattern already adopted by any other service messages, it seems to me, they all use `msg_id` as a field for representing ID.

Reply:
Yes, this could be a target for a future refactor to disambiguate but from what I understand, msg_id helps track the original message between being boxed and unboxed while preventing confusion with other ids used in protos like:

target_id
request_id
location_id
id_at_location

```python
# message.py

class SyftMessage(AbstractMessage):
    def __init__(self, address: Address, msg_id: Optional[UID] = None) -> None:
        self.address = address
        super().__init__(id=msg_id)

...

return SignedMessage_PB(
    msg_id=self.id.proto(),

```